modules/local/spades.nf

```diff
@@ -20,14 +20,15 @@ process SPADES {
     script:
     def args = task.ext.args ?: ''
     maxmem = task.memory.toGiga()
+    def readstr = meta.single_end ? "--12 ${reads}" : "-1 ${reads[0]} -2 ${reads[1]}"
```

Comment:
```suggestion
    def readstr = meta.single_end ? "--12 ${reads}" : "-pe1-1 ${reads[0]} -pe1-2 ${reads[1]}"
```
Or is there a reason not to use "pe"? (I might be not up to date with SPAdes usage)